### DISCUSSION TITLE
Standard tweaks disables RDP?

### DISCUSSION THREAD

**Discussion by Vassterak** [ChrisTitusTech/winutil#2093] [Q&A] [answered]
Hi,
Today I've tried running tweaks on 2 remote machines I've logged in using RDP (local network only).
And after running **"Standard tweaks"** and waiting for disk cleanup to end. I disconnected from RDP and couple minutes later I wanted to reconnect but now I'm unable to do that. **Is it that "Set services to manual" disables RDP listening for incoming connections?**
I would appreciate that RDP service could be opt out from the tweaks! Thanks.
I know that it's my mistake and I didn't realized that RDP service will be also set to manual...

**Comment by Marterich** [comment, marked as answer]
This is due to a Change in the "Recommended" Tweaks by OO-Shutup. 
OO-Solutions keeps changing its recommendations every now and then and to keep winutil maintainable we opted to apply all OO recommendations without any customization. 

To undo the change: 
1. Click "Customize OO Shutup Tweaks" under Advanced Tweaks
2. Click on the "Local Machine" Tab
3. Search for "Disable remote connections to this computer"
4. Disable the tweak
 
Afterwards, you should be all set :)

_Originally posted by @Marterich in https://github.com/ChrisTitusTech/winutil/issues/2082#issuecomment-2163776589_

Addition: The OOSU checkbox will most likely be removed in the #2099 to prevent this kind of issues in the future

**Reply by Vassterak** [reply]
Ok, thank you.
Sorry I totally overlooked it in 'Issues'. I probably searched it using "Remote Desktop" and it didn't show up.

**Reply by Marterich** [reply]
You're welcome :) if all is good, you can mark this message as an answer so it's easier for others to find
